### PR TITLE
Fix i18n in HTML Files, Part Two

### LINKED ISSUE #8904: Bulk internationalization of HTML title/values where needed
### Problem

While I'm on an internationalization kick, I've noticed that there are a number of untranslated HTML `title` and `value` attributes (the value is often used to show button text) -- it would very simple to search the codebase for these, and to fix all that can be found in one go. 

#### Evidence / Screenshot
**Sample untranslated `title`:**
https://github.com/internetarchive/openlibrary/blob/d8569549d75da79d156323740fed79128fc92683/openlibrary/templates/history.html#L53-L60
**Sample untranslated on-screen `value`:**
https://github.com/internetarchive/openlibrary/blob/d8569549d75da79d156323740fed79128fc92683/openlibrary/macros/ManageWaitlistButton.html#L38
**Other untranslated on-screen text (fix can be bundled in):**
https://github.com/internetarchive/openlibrary/blob/d8569549d75da79d156323740fed79128fc92683/openlibrary/macros/ManageWaitlistButton.html#L42

### Reproducing the bug

1. Search the codebase for title=" and value=" 
2. A number of titles and on-screen values such as button text will show up improperly formatted for i18n

* Expected behavior: All on-screen text should be formatted like so: https://github.com/internetarchive/openlibrary/blob/d8569549d75da79d156323740fed79128fc92683/openlibrary/macros/EditButtonsMacros.html#L23
* Actual behavior:  A good amount of it is formatted like the examples shown above.

### Context

_No response_

#### Proposal & constraints

Proposed solution is to go through and manually update each of the titles/values for i18n. Will be a somewhat repetitive task, but it's a very easy fix!

Helpful resources: 
* [i18n Guidelines](https://github.com/internetarchive/openlibrary/wiki/Internationalization-Contributor's-Guide-(i18n)#internationalization-i18n-developers-guide)
* [.pot File Generation Guidelines](https://github.com/internetarchive/openlibrary/wiki/Internationalization-Contributor's-Guide-(i18n)#extracting-strings-from-htmlpython-files-generating-the-pot-file) -- would be helpful to run this after the changes have been made to help keep the translations up to date

### PR DESCRIPTION
**Pull request by merwhite11**
Closes #8904 

Refactor. Added i18n syntax for value, title and alt attributes in remaining HTML files. 

### Technical

Ran extraction command, no failures, messages.pot file updated. 

### Testing

**General**
`docker compose run web make test`
`docker compose run web pytest openlibrary/plugins/importapi/tests/test_import_validator.py`

**To extract syntaxed messages:**
```
docker compose build home (to incorporate updates to docker file)
docker compose run --rm -uroot home ./scripts/i18n-messages extract
```
### Stakeholders

@rebecca-shoptaw 
@cdrini

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,6 +119,8 @@
     "collectCoverage": true
```
Comment (cdrini) [inline package.json:119]: Oh these snuck in and shouldn't be here! You can revert them back by doing something like:

```
# First update your master branch
# then,
git checkout master package-lock.json
git checkout master package.json
```

This should take the version of these files currently on master, effectively "undoing" the changes in your branch! Save these on a separate commit :+1:

File: openlibrary/templates/lists/list_overview.html
Comment (merwhite11) [inline openlibrary/templates/lists/list_overview.html:1]: I just updated this to follow this syntax: openlibrary/templates/books/daisy.html
But neither this message or the one from daisy.html are making it to messages.pot

Comment (merwhite11) [inline openlibrary/templates/lists/list_overview.html:1]: Not sure if this is valid. Modeled after openlibrary/templates/books/daisy.html and neither messages are showing up in messages.pot

File: openlibrary/templates/books/mobile_carousel.html [hunk 2]
```diff
@@ -49,8 +49,8 @@
       <a class="mybooks mobile nopadding" href="$(url)">
         $if cover_url:
           <img class="bookcover mobile $(last_book)" loading="lazy"
-            title="$('%s%s'%(title,byline))"
-            alt="$('%s%s'%(title,byline))"
+            title="$_('%(title)s%(byline)s', title=title, byline=byline)"
+            alt="$_('%(title)s%(byline)s', title=title, byline=byline)"
```
Comment (cdrini) [inline openlibrary/templates/books/mobile_carousel.html:53]: Ah there's nothing to actually translate here! So can probably just replace with:

```suggestion
            title="$title$byline"
            alt="$title$byline"
```

**Review by cdrini — COMMENTED** [review]
Nice looks good! A few small fixes :+1:

**Comment by cdrini** [comment]
Ok I also cleaned up those package*.json files:

```sh
$ git fetch upstream master
$ git checkout upstream/master package-lock.json
Updated 1 path from fd852cf6f
$ gitpod /workspace/openlibrary (#8904-Second-Batch) $ git checkout upstream/master package.json
Updated 1 path from fd852cf6f
$ git status
On branch #8904-Second-Batch
Your branch is up to date with 'origin/#8904-Second-Batch'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
        modified:   package-lock.json
        modified:   package.json

$ git commit -m "Revert unintentional changes to package*.json files"
[#8904-Second-Batch cc6598a8f] Revert unintentional changes to package*.json files
 2 files changed, 2 insertions(+), 2 deletions(-)
$ git push origin HEAD
Enumerating objects: 7, done.
Counting objects: 100% (7/7), done.
Delta compression using up to 16 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 427 bytes | 427.00 KiB/s, done.
Total 4 (delta 3), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
To https://github.com/merwhite11/openlibrary.git
   80002ad13..cc6598a8f  HEAD -> #8904-Second-Batch
gitpod /workspace/openlibrary (#8904-Second-Batch) $ 
```

File: openlibrary/templates/lists/list_overview.html
Comment (cdrini) [reply openlibrary/templates/lists/list_overview.html:1]: Hmmm that's odd! Can you create a new issue for this? There might be a bug preventing certain files from being read!

Comment (merwhite11) [reply openlibrary/templates/lists/list_overview.html:1]: @cdrini 
I checked the pot file and the 'from You' seems to be extracting fine now that it's not an if/else --
https://github.com/internetarchive/openlibrary/blob/e52c6804785ecf8a565466637184d0cb56b73399/openlibrary/i18n/messages.pot#L4421

Comment (cdrini) [reply openlibrary/templates/lists/list_overview.html:1]: Oh sweet! Woohoo closed an issue before it was even opened! 😁 :success-kid:

**Pull request merged**